scripts/start.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+yarn relay keys & 
```

Comment:
This doesn't solve the problems i mentioned before. It's the same command just split into many lines.

We need a nodejs process that is managing child processes. The approach now starts a background process but doesn't provide a process ID. This approach will create a complicated terminal state if anything goes wrong.

Check [here](https://github.com/Unirep/Unirep/blob/main/packages/circuits/scripts/buildSnarks.mjs) to see how i invoke command line stuff inside nodejs. I think it's a decent idea to have a single process managing each of these commands as subprocesses, we just have to build a robust solution if we do this, otherwise it will confuse devs more.